versions.tf

```diff
@@ -6,7 +6,7 @@ terraform {
     time   = "~> 0.6"
     lacework = {
       source  = "lacework/lacework"
-      version = "~> 0.2"
```

Comment:
Hey @mikelaramie, why is the minimum provider version being changed here?

Reply:
I guess it's a superfluous change since we're going to pull 0.15 or later regardless; but I wanted to make sure anything I leveraged in here wasn't something relatively new. I did not test this code on anything besides 0.15

Reply:
Rolled back to 0.2